test/NuGet.Core.Tests/NuGet.Build.Tasks.Pack.Test/PackTaskLogicTests.cs

```diff
@@ -9,7 +9,6 @@
 using NuGet.Frameworks;
```

Comment:
Can you please add an E2E pack test? 

https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs is probably where you want that in.

Reply:
Added - however it is disabled now. See issue: https://github.com/NuGet/Home/issues/9400

Reply:
Enabled and using Directory.Build.props for now.